### ISSUE TITLE
Remove HMR_4362 from the model

### ISSUE THREAD

**Issue by avlant** [SysBioChalmers/Human-GEM#153]
### Description of the issue:
The following reaction HMR_4362 (ATP[c] + H2O[c] + pyruvate[c] => AMP[c] + PEP[c] + Pi[c]) with EC number 2.7.9.2 is not supported by a reference in the model, and does not seem to exist in human according to KEGG [1], Brenda [2] or in textbook descriptions of gluconeogenesis [3, 4]. 

[1] https://www.genome.jp/kegg-bin/show_pathway?org_name=hsa&mapno=00620&mapscale=&show_description=hide
[2] https://www.brenda-enzymes.org/enzyme.php?ecno=2.7.9.2
[3] https://socratic.org/questions/how-is-gluconeogenesis-related-to-glycolysis
[4] https://themedicalbiochemistrypage.org/gluconeogenesis.php

#### Expected feature/value/output:
The reaction should be removed.

#### Current feature/value/output:
It exists.

**Comment by Hao-Chalmers** [comment]
@avlant thanks for posting the issue.

If you happen to know the reason why this reaction was included initially, maybe that would help in avoiding similar problems in the future.

**Comment by JonathanRob** [comment]
Addressed in #171.

**Comment by mihai-sysbio** [comment]
@all-contributors please include @avlant for bug and content

**Comment by mihai-sysbio** [comment]
Alright, let me try again.

@all-contributors please add @avlant for bug and content
